src/applications/static-pages/sass/modules/_m-facility-detail.scss

```diff
@@ -1,5 +1,5 @@
 .va-c-facility-hours-list {
-   list-style-type: none;
+   list-style-type: none !important;
```

Comment:
Is the `!important` really needed here? It's better to avoid it if possible.

Reply:
Yeah. Wasn't able to get around it.

Reply:
I think you can add `class="usa-unstyled-list"` to the `<ul>` instead of adding this rule